src/js/genomic.js

```diff
@@ -0,0 +1,84 @@
+var d3 = require('d3');
+var _ = require('underscore');
+
+var renderers = require('./renderers');
+var core = require('./core');
+var utils = require('./utils');
+
+var oncoprint_core = core();
+
+var config = { rect_height: 20,
+              rect_padding: 3,
+              rect_width: 10,
+              mutation_fill: 'green',
+
+              cna_fills: {
+              null: 'grey',
+              undefined: 'grey',
+              AMPLIFIED: 'red',
+              HOMODELETED: 'blue'
+             }
+};
+
+var gene_renderer = renderers.gene(config);
+
+function is_sample_genetically_altered(datum) {
+  return datum.cna !== undefined
+    || datum.mutation !== undefined
+    || datum.rna !== undefined
+    || datum.protein !== undefined;
+};
+
+function row_to_labels(row) {
```

Comment:
Sorry which one? I don't seem to be able to see the specific line that you commented on. Are you talking about `row_to_labels`? I renamed it `calculate_row_label`..